mmv1/products/dataform/Repository.yaml

```
@@ -25,12 +25,18 @@ references: !ruby/object:Api::Resource::ReferenceLinks
     'Official Documentation': 'https://cloud.google.com/dataform/docs/'
   api: 'https://cloud.google.com/dataform/reference/rest/v1beta1/projects.locations.repositories'
 id_format: projects/{{project}}/locations/{{region}}/repositories/{{name}}
+iam_policy: !ruby/object:Api::Resource::IamPolicy
+  method_name_separator: ':'
+  parent_resource_attribute: 'repository'
+  min_version: beta
 import_format:
   ['projects/{{project}}/locations/{{region}}/repositories/{{name}}']
 examples:
   - !ruby/object:Provider::Terraform::Examples
     name: 'dataform_repository'
     primary_resource_id: dataform_respository
+    primary_resource_name:
+      'fmt.Sprintf("tf_test_repository%s", context["random_suffix"])'
```

Looks like this is the cause of the confusion - the acceptance test creates a resource with the name `tf_test_dataform_repository<suffix>` but then tried to import IAM resources on a `tf_test_repository<suffix>` resource.

With the suggested change below I've had the tests pass locally: 

```suggestion
      'fmt.Sprintf("tf_test_dataform_repository%s", context["random_suffix"])'
```

---

I went ahead and made the commit so we can get the tests started - once they pass I'll approve and merge!

---

Thank You